Homework_6/Homew.py

```diff
@@ -0,0 +1,59 @@
+import os
+import re
+import csv
+
+CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+def collect_data():
+    data_dir = os.path.join(CURRENT_DIR, 'source_data')
+    result = []
+    source_files = [i for i in os.listdir(data_dir) if i.split('.')[1] == 'txt']
+
+    for filename in source_files:
+        filepath = os.path.join(data_dir, filename)
+
+        with open(filepath) as fl:
```

Comment:
You must add encoding.
But first you must detect it, otherwise Linux and MacOS users will not be able to run your script.